### PR TITLE
login form initial commit

### PR DESCRIPTION
**Pull request by ganesh35palkar**

### CODE REVIEW

File: soriuser/admin.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from soriuser.models import UserLogin
```
Comment (ganesh35palkar) [inline soriuser/admin.py]: what is UserLogin?? Is it necessary to use

File: soriuser/forms.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from django import forms
+from django.contrib.auth.models import User
+from rango.models import UserLogin
```
Comment (ganesh35palkar) [inline soriuser/forms.py]: If you have django.contrib.auth.models.User then do we need UserLogin, and why rango.models ??

File: soriuser/forms.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+class LoginForm(forms.ModelForm):
```
Comment (ganesh35palkar) [inline soriuser/forms.py]: need two blan lines before this line

File: soriuser/models.py [hunk 4]
```diff
@@ -1,3 +1,8 @@
 from django.db import models
+from django.contrib.auth.models import User 
 
-# Create your models here.
+class UserLogin(models.Model):
```
Comment (ganesh35palkar) [inline soriuser/models.py]: we dont need UserLogin model

File: soriuser/views.py [hunk 5]
```diff
@@ -1,3 +1,42 @@
 from django.shortcuts import render
+from django.http import HttpResponse, HttpResponseRedirect
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+def index(request):
```
Comment (ganesh35palkar) [inline soriuser/views.py]: we do not need index view here

File: templates/soriuser/homepage.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+	{% if user.is_authenticated %}
+		<h1>Shoffex welcomes you, {{user.username}}!</h1>
+	{% else %}
+		<h1>Please Login to continue</h1>
+	{% endif %}
+	<li>
+		<a href="/soriuser/logout/">Logout</a>
+	</li>
+</body>
+</html>
```
Comment (ganesh35palkar) [inline templates/soriuser/homepage.html]: add blank line at the end of file

File: templates/soriuser/index.html [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+	<title>Shoffex.com</title>
+</head>
+<body>
+	<h1>Shoffex <h4>shop with LOVE</h4></h1>
+	<li>
+		<a href="/soriuser/login/">Login</a>
+	</li>
+</body>
+</html>
```
Comment (ganesh35palkar) [inline templates/soriuser/index.html]: add blank line at the end of file

File: templates/soriuser/login.html [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+		Username:<input type="text" name="username" value="" size="50" />
+		<br/>
+		Password:<input type="password" name="password" value="" size="50" />
+		<br/>
+
+		<input type="submit" value="submit" />
+	</form>
+
+</body>
+</html>
```
Comment (ganesh35palkar) [inline templates/soriuser/login.html]: add blank line at the end of file

Comment (rajshoffex) [inline templates/soriuser/login.html]: i have unnecessarily added UserLogin model, but then we dont have to add
any code in model for the Login page task, will it be ok that we don't use
any model for this purpose

Comment (rajshoffex) [inline templates/soriuser/login.html]: I will make the changes
 suggested by you, is there any addition, or any changes to do???

Comment (rajshoffex) [inline templates/soriuser/login.html]: i got links to read, rest yesterday's work is ok or not???

**Pull request merged**